tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -139,24 +139,24 @@ - (id)init {
 
 - (BOOL)application:(UIApplication*)application willFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
     ASSERT_TRUE(launchOptions[UIApplicationLaunchOptionsVoiceCommandKey]);
-    WMSSpeechRecognitionResult* result = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
-    ASSERT_STREQ("CORTANA_TEST", [result.text UTF8String]);
+    WAAVoiceCommandActivatedEventArgs* args = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
+    ASSERT_STREQ("CORTANA_TEST", [args.result.text UTF8String]);
     _methodsCalled[NSStringFromSelector(_cmd)] = @(YES);
     return true;
 }
 
 - (BOOL)application:(UIApplication*)application didFinishLaunchingWithOptions:(NSDictionary*)launchOptions {
     ASSERT_TRUE(launchOptions[UIApplicationLaunchOptionsVoiceCommandKey]);
-    WMSSpeechRecognitionResult* result = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
-    ASSERT_STREQ("CORTANA_TEST", [result.text UTF8String]);
+    WAAVoiceCommandActivatedEventArgs* args = launchOptions[UIApplicationLaunchOptionsVoiceCommandKey];
+    ASSERT_STREQ("CORTANA_TEST", [args.result.text UTF8String]);
     _methodsCalled[NSStringFromSelector(_cmd)] = @(YES);
     return true;
 }
 
-- (BOOL)application:(UIApplication*)application didReceiveVoiceCommand:(WMSSpeechRecognitionResult*)result {
+- (BOOL)application:(UIApplication*)application didReceiveVoiceCommand:(WAAVoiceCommandActivatedEventArgs*)args {
     // Delegate method should only be called once
     ASSERT_EQ([[self methodsCalled] objectForKey:NSStringFromSelector(_cmd)], nil);
-    ASSERT_STREQ("CORTANA_TEST", [result.text UTF8String]);
+    ASSERT_STREQ("CORTANA_TEST", [args.result.text UTF8String]);
```

Comment:
>ASSERT_STREQ [](start = 4, length = 12)

While you're changing this, you could just use `ASSERT_OBJCEQ(@"CORTANA_TEST", args.result.text);`